examples/imagenet/train_benchmark.py

```diff
@@ -43,7 +43,6 @@ def test_8x_v100_half_precision(self):
     model_dir = tempfile.mkdtemp()
     FLAGS.batch_size = 2048
     FLAGS.half_precision = True
-    FLAGS.loss_scaling = 256.
```

Comment:
I know that it's technically not a part of your PR, but you could also update the README fo the imagenet example? It currently lists `--loss_scaling=256.` as a command line argument.

Reply:
Good point. Thanks for catching this.